components/paytrace/actions/batch-summary/batch-summary.mjs

```diff
@@ -0,0 +1,35 @@
+import { axios } from "@pipedream/platform"
```

Comment:
Can you please take a look at [this component](https://github.com/PipedreamHQ/pipedream/blob/master/components/segment/segment.app.mjs) as an example and make use also of `paytrace.app.mjs` file? In that way you can centralize the axios import in that file and handle better all needed endpoints in only one place?